### PR TITLE
Migrate Strip to TypeScript

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Migrate Strip to TypeScript.

## Fixes

Fixes: #229.

### CODE REVIEW

File: src/components/Strip/Strip.tsx [hunk 1]
```diff
@@ -1,26 +1,44 @@
...
+  colSize?: ColSize;
+  dark?: boolean;
+  deep?: boolean;
+  element?: ElementType;
+  includeCol?: boolean;
+  light?: boolean;
+  rowClassName?: string;
+  shallow?: boolean;
+  type?: string;
+} & HTMLProps<HTMLElement>;
```
Comment (Caleb-Ellis) [inline src/components/Strip/Strip.tsx:24]: Forgot about this in the other PRs but we should add descriptions for each prop

Comment (huwshimi) [reply src/components/Strip/Strip.tsx:24]: Yeah good idea, might have to go back over the other components. Also might be worth trying to generate the prop types once they're all migrated: https://github.com/canonical-web-and-design/react-components/issues/201.

Comment (huwshimi) [reply src/components/Strip/Strip.tsx]: I've pushed these.

**Review by Caleb-Ellis — APPROVED** [review]
LGTM!

**Pull request merged**